cabot/cabotapp/migrations/0026_auto__add_field_instance_hipchat_room_id__add_field_service_hipchat_ro.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'Instance.hipchat_room_id'
+        db.add_column(u'cabotapp_instance', 'hipchat_room_id',
```

Comment:
Looks like instances by default have hipchat alerts enabled, so if send_alert() is called on the instance, `service.hipchat_room_id` in cabot_alert_hipchat/models.py would throw an exception (in this case `service` is actually an instance :( ). I could have a try/catch for that or just also have a hipchat_room_id for instance like this, even if it's always null